docs/view_components/welcome.rst

```diff
@@ -1,11 +1,47 @@
-Component title
----------------
+.. _welcome-label:
+
+Welcome page
+------------
+
+The welcome window serves as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+
 
 Attributes
 ^^^^^^^^^^
 
+#. **Welcome message**
+    * A text
+
+#. **Video intro message**
+    * A short, 3 minute video clip that provides the important information such as tool features, capabilities, where to find help, etc., very briefly
+    * This clip is embedded into the welcome pop-up. The user can view it on YouTube (or Vimeo) on a separate tab of the browser by clicking on a link on the video
+
+#. **Check-box to disable the welcome pop-up**
+
+#. **Hyperlinks**
+    * Clickable links to read more, documentation, examples, use-cases, video tutorials, code-base (GitHub repo) and contact us
+
 Actions
 ^^^^^^^
 
+* Pop-up that springs up everytime the software is started
```

Comment:
I think this belong to the requirements, because the user cannot act on the fact that the welcome view-component will be visible the first time they open the tool